### PR TITLE
openrct2-ui C++

### PR DESCRIPTION
**Pull request by janisozaur**

### CODE REVIEW

File: src/openrct2-ui/CMakeLists.txt [hunk 1]
```diff
@@ -25,7 +25,6 @@ endif ()
 
 # Sources
 file(GLOB_RECURSE OPENRCT2_UI_SOURCES
-    "${CMAKE_CURRENT_LIST_DIR}/*.c"
```
Comment (IntelOrca) [inline src/openrct2-ui/CMakeLists.txt]: Mirror to vcxproj?

**Comment by Gymnasiast** [comment]
Fixed the Xcode project and also renamed input.h to Input.h, which you forgot.

File: src/openrct2-ui/input/KeyboardShortcut.cpp [hunk 2]
```diff
@@ -14,32 +14,36 @@
...
 #include "KeyboardShortcuts.h"
 
 uint8 gKeyboardShortcutChangeId;
 
 typedef void (*shortcut_action)();
 
-static const shortcut_action shortcut_table[SHORTCUT_COUNT];
+namespace
+{
+extern const shortcut_action shortcut_table[SHORTCUT_COUNT];
```
Comment (Gymnasiast) [inline src/openrct2-ui/input/KeyboardShortcut.cpp]: Should this not be indented?

**Comment by Gymnasiast** [comment]
Can you fix the alignment bit, so this can be merged?

File: src/openrct2-ui/input/KeyboardShortcut.cpp [hunk 3]
```diff
@@ -671,78 +730,82 @@ static void shortcut_ride_construction_demolish_current()
...
-    shortcut_ride_construction_previous_track,
-    shortcut_ride_construction_next_track,
-    shortcut_ride_construction_build_current,
-    shortcut_ride_construction_demolish_current,
-    shortcut_load_game,
-    shortcut_clear_scenery,
-};
+namespace
+{
+    const shortcut_action shortcut_table[SHORTCUT_COUNT] = {
```
Comment (Gymnasiast) [inline src/openrct2-ui/input/KeyboardShortcut.cpp]: Brace should go on new line.
(Seems to be the last formatting issue.)

**Pull request merged**